tests/auth0/handlers/roles.tests.js

```diff
@@ -269,18 +281,17 @@ describe('#roles handler', () => {
             expect(data.id).to.equal('myRoleId');
             return Promise.resolve(data);
           },
-          getAll: () => [
-            {
-              name: 'myRole',
-              id: 'myRoleId',
-              description: 'myDescription',
-              permissions: [
-                {
-                  permission_name: 'Create:cal_entry', resource_server_identifier: 'organise'
-                }
-              ]
-            }
-          ],
+          getAll: () => Promise.resolve({
+            roles: [
+              {
+                name: 'myRole',
+                id: 'myRoleId',
+                description: 'myDescription'
+              }
+            ],
+            total: 1,
+            limit: 50
+          }),
```

Comment:
Would it be possible to add a multi-page test case to make sure all the paging logic gets exercised?

Reply:
sure! it's a very good idea